### ISSUE TITLE
Change browser tab labels for Live Feeds display

### ISSUE THREAD

**Issue by MadokVaur** [mastodon/mastodon#27215]
### Pitch

Applies to Firefox on Ubuntu Linux desktop

My instance went to v4.2.0 over night. Checking out the right-hand sidebar Live Feeds where we see:

This Server [pubic/local], Other Servers [public/remote] and All [public]

All three of these tabs are labeled "Live Feeds - [instance_name]" so to look at the tab bar they are indistinguishable as to what they really are

Proposal: relabel web browser tabs to more accurately identify contents: Local, Other, Pubic perhaps

### Motivation

Again, applies to Firefox on Ubuntu tabs only. I don't know about other web browsers, let alone how any apps are handling this. But a label change might be a start to clarifying those too.

**Comment by ClearlyClaire** [comment]
Ah, having a more specific title would make sense indeed!

Not sure whether we should keep the old terminology (“Local timeline” and “Federated timeline”) or come up with something new to improve consistency with the “Live Feeds” and “This server” / “Other servers” / “All” terminology

**Comment by MadokVaur** [comment]
Yeah, I honestly don't know how many people have multiple browser tabs open at once [waves hand] but labeling the individual tabs might be helpful anyway down the line

Right now I'm trying to understand the different feeds as to content. 

Remote [public/remote] seems like it might be the old Federated feed, moderated/curated by my instance admins (or someone) somehow. 

All [public] seems to be an unmoderated/curated feed like the old unexpurgated Federate complete with all the pr0n...

**Comment by ClearlyClaire** [comment]
No, nothing is curated, it's like before. “This server” is what “Local timeline” was, “Remote” is what “Federated timeline” was when the option to exclude local posts was enabled, and “All” is what “Federated timeline” was otherwise.

**Comment by MadokVaur** [comment]
Ah...

Thank you for that clarification. Much appreciated!
